packages/flight-icons/catalog.json

```diff
@@ -1920,7 +1956,7 @@
       "height": 16
     },
     {
-      "id": "855:190",
+      "id": "4425:94",
       "fileName": "twitter-24",
       "iconName": "twitter",
       "description": "twitter",
```

Comment:
I'm curious about is whether the description should also be updated for these icons. It would probably be useful to include "x" even though that might increase the noise when searching.

Reply:
Ah, yeah I see what you mean, I could adjust the naming to `twitter, x`.

Reply:
Right, which may increase it's discoverability, but will also surface other icons (x-hexagon, x-square, etc)

Reply:
Hmm, I feel that leaving it as `twitter` shouldn't divert the searching experience too much for consumers since the brand identity change is still fresh with most users, at least from what i've seen when people discuss `twitters` brand/name change. So keeping as is should be okay, and in return decrease the extra `x` related searches. wdyt?